x-pack/plugins/security_solution/public/overview/components/recent_cases/recent_cases.tsx

```diff
@@ -52,7 +52,7 @@ const RecentCasesComponent = ({ cases }: { cases: Case[] }) => {
             {c.description && c.description.length && (
               <MarkdownContainer>
                 <EuiText color="subdued" size="xs">
-                  <Markdown disableLinks={true} raw={c.description} />
+                  <MarkdownRenderer>{c.description}</MarkdownRenderer>
```

Comment:
Please pass an equivalent to the `disableLinks={true}` prop here to disable hyperlinks in the descriptions of cases that appear on the Overview page.